src/game/logic/scriptengine/scriptengine.cpp

```diff
@@ -13,56 +13,9407 @@
+    m_actionTemplates[ScriptAction::CAMERA_SET_AUDIBLE_DISTANCE].m_uiStrings[0] =
+        "Set the audible range during camera-up shots to ";
+
+    m_actionTemplates[ScriptAction::NAMED_SET_HELD].m_internalName = "NAMED_SET_HELD";
+    m_actionTemplates[ScriptAction::NAMED_SET_HELD].m_uiName =
+        "Unit_/Move/Set unit to be held in place, ignoring Physics, Locomotors, etc.";
+    m_actionTemplates[ScriptAction::NAMED_SET_HELD].m_numParameters = 2;
+    m_actionTemplates[ScriptAction::NAMED_SET_HELD].m_parameters[0] = Parameter::UNIT;
+    m_actionTemplates[ScriptAction::NAMED_SET_HELD].m_parameters[1] = Parameter::BOOLEAN;
+    m_actionTemplates[ScriptAction::NAMED_SET_HELD].m_numUiStrings = 3;
+    m_actionTemplates[ScriptAction::NAMED_SET_HELD].m_uiStrings[0] = "Set Held status for ";
+    m_actionTemplates[ScriptAction::NAMED_SET_HELD].m_uiStrings[1] = " to ";
+    m_actionTemplates[ScriptAction::NAMED_SET_HELD].m_uiStrings[2] = ".";
+
+    m_actionTemplates[ScriptAction::NAMED_SET_STOPPING_DISTANCE].m_internalName = "NAMED_SET_STOPPING_DISTANCE";
+    m_actionTemplates[ScriptAction::NAMED_SET_STOPPING_DISTANCE].m_uiName =
+        "Unit_/Move/Set stopping distance for current locomotor.";
+    m_actionTemplates[ScriptAction::NAMED_SET_STOPPING_DISTANCE].m_numParameters = 2;
+    m_actionTemplates[ScriptAction::NAMED_SET_STOPPING_DISTANCE].m_parameters[0] = Parameter::UNIT;
+    m_actionTemplates[ScriptAction::NAMED_SET_STOPPING_DISTANCE].m_parameters[1] = Parameter::REAL;
+    m_actionTemplates[ScriptAction::NAMED_SET_STOPPING_DISTANCE].m_numUiStrings = 3;
+    m_actionTemplates[ScriptAction::NAMED_SET_STOPPING_DISTANCE].m_uiStrings[0] = "Set stopping distance for ";
+    m_actionTemplates[ScriptAction::NAMED_SET_STOPPING_DISTANCE].m_uiStrings[1] = " to ";
+    m_actionTemplates[ScriptAction::NAMED_SET_STOPPING_DISTANCE].m_uiStrings[2] = ".";
+
+    m_actionTemplates[ScriptAction::SET_STOPPING_DISTANCE].m_internalName = "SET_STOPPING_DISTANCE";
+    m_actionTemplates[ScriptAction::SET_STOPPING_DISTANCE].m_uiName =
+        "Team_/Move/Set stopping distance for each unit's current locomotor.";
+    m_actionTemplates[ScriptAction::SET_STOPPING_DISTANCE].m_numParameters = 2;
+    m_actionTemplates[ScriptAction::SET_STOPPING_DISTANCE].m_parameters[0] = Parameter::TEAM;
+    m_actionTemplates[ScriptAction::SET_STOPPING_DISTANCE].m_parameters[1] = Parameter::REAL;
+    m_actionTemplates[ScriptAction::SET_STOPPING_DISTANCE].m_numUiStrings = 3;
+    m_actionTemplates[ScriptAction::SET_STOPPING_DISTANCE].m_uiStrings[0] = "Set stopping distances for ";
+    m_actionTemplates[ScriptAction::SET_STOPPING_DISTANCE].m_uiStrings[1] = " to ";
+    m_actionTemplates[ScriptAction::SET_STOPPING_DISTANCE].m_uiStrings[2] = ".";
+
+    m_actionTemplates[ScriptAction::SET_FPS_LIMIT].m_internalName = "SET_FPS_LIMIT";
+    m_actionTemplates[ScriptAction::SET_FPS_LIMIT].m_uiName = "Scripting_/ Set max frames per second.";
```

Comment:
Beginning here, many ui names suddenly have a whitespace after the /. Is this proper?

Reply:
This is correct.